index.js

```diff
@@ -163,6 +164,13 @@ RedirectableRequest.prototype._processResponse = function (response) {
   // referenced by the Location field value,
   // even if the specific status code is not understood.
   var location = response.headers.location;
+  if (this._options.trackRedirects === true) {
+    this._redirects.push({
+      "url": this._currentUrl,
+      "headers": response.headers,
+      "statusCode": response.statusCode
+    });
```

Comment:
Please unquote all of these keys and add a dangling comma to the last. (Linter is failing.)